taxi/models.py

```diff
@@ -15,7 +16,25 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    @staticmethod
+    def validate_license_number(value: str) -> None:
```

Comment:
validation needs to be added on forms, try using built-in Validators there to simplify code